smithy-aws-iam-traits/src/main/resources/META-INF/smithy/aws.iam.smithy

```diff
@@ -46,6 +71,7 @@ structure iamResource {
 }
 
 /// Other actions that the invoker must be authorized to perform when executing the targeted operation.
+@deprecated(since: "2023-10-31", message: "Use the `requiredActions` member of the `aws.iam#iamAction` trait instead.")
```

Comment:
Why would the deprecated since be Oct 31? Wouldnt the since date be the day we add deprecated trait and merge commit?

Reply:
10/31 is when I wrote the deprecation traits for this work. I can adjust it to a target date for the next release.